src/version.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module ckeditor5/version
+ */
+
+/* global window, global */
+
+import CKEditorError from '@ckeditor/ckeditor5-utils/src/ckeditorerror';
+import { version } from '../package.json';
+
+const windowOrGlobal = window || global;
+
+if ( windowOrGlobal.CKEDITOR_VERSION ) {
+	/**
+	 * This error is thrown when the base {@link module:core/editor/editor~Editor `Editor`}
+	 * class has been loaded twice (or more), a result of either:
+	 *
+	 * * the build containing two (or more) different versions of the {@link module:core/editor/editor~Editor `Editor`}
+	 * class – make sure **all** packages in your {@glink builds/index build} which have
+	 * `@ckeditor/ckeditor5-core` in their `package.json` [dependencies](https://docs.npmjs.com/files/package.json#dependencies)
+	 * use the same version of the package,
+	 * * two (or more) `ckeditor.js` builds loaded in the web page next to one another – check your web page for duplicated
+	 * `<script>` elements or make sure your page builder/bundler includes CKEditor only once.
+	 *
+	 * @error ckeditor-version-collision
+	 * @param {String} collidingVersion The version of the build which has already been (incorrectly) loaded.
+	 * @param {String} version The version of the build which is supposed to be loaded.
+	 */
+	throw new CKEditorError( 'ckeditor-version-collision: The global CKEDITOR_VERSION constant has already been set.', {
+		collidingVersion: windowOrGlobal.CKEDITOR_VERSION,
+		version
+	} );
```

Comment:
I would throw only if the version is different. If the version is the same, but it is loaded multiple times there should be a (big) warning in the console. Otherwise, we make the editor stop working in some cases when people try to prototype something. And, even though, such prototyping will not work in some cases (because of https://github.com/ckeditor/ckeditor5/issues/667) we will make it stop working in all cases. In other words, loading this module twice does not necessarily mean that something will be broken I think that warning will be much better.

Reply:
cc @Reinmar

Reply:
Right now the editor doesn't work anyway when things are duplicated (we're talking here about serious duplication of the entire `Editor` module) so an error makes a lot of sense. We may change it once we'll fix the situation.

Reply:
Alternatively, we can change it into `console.error()`.

Reply:
`log.error()` then?

Reply:
`log.error()` is not available here, unfortunately. That'd mean that `ckeditor5` depends `ckeditor5-utils`. 

We could depend on `ckeditor5-utils` if all this code was in `ckeditor5-core`. Maybe we should go this way?

PS. Using `log.error()` is better because we'll also log a link to docs.ckeditor.com.

Reply:
Maybe let's move the `version.js` to utils?

Reply:
And what would be left here, in `ckeditor5`? It'd be empty, yes? So this change would be limited to `ckeditor5-core` and `ckeditor5-utils`?

Reply:
Yes.

Reply:
Let's do this, then.